src/lib.rs

```diff
@@ -56,7 +56,7 @@
 //! ```
 //!
 //! ## Extra information
-//! You can add arbitrary extra information using LocatedSpanEx.
+//! You can add arbitrary extra information using `LocatedSpanEx`. This extra property is not used when comparing two `LocatedSpanEx`s.
```

Comment:
newline please

Reply:
Sure! Are you trying to wrap comment lines at 80 characters or do you simply want a newline between the sentences?

Reply:
Both are ok

Reply:
Okay, I've just split the sentences on to newlines as the rest of the comments are not wrapped at 80 characters anyways